gui/resources.xml

```diff
@@ -597,21 +618,21 @@ Vocabulary
         <object class="GtkAction" id="ResetZoom">
           <property name="icon-name">mypaint-view-100-symbolic</property>
           <property name="label" translatable="yes" context="Menu→View→Reset (labels), Accel Editor (labels)">Reset Zoom</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Restore the view’s zoom to its default.</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Restore the view’s zoom to 100%.</property>
```

Comment:
I'm a bit confused about why the word 'its' -- which makes the whole phrase awkward -- is in there. We can't reset the zoom of the scratchpad using this action, and it also suggests that individual views may have different defaults (which would be a Bad Idea. Thankfully we aren't currently doing it.)
.. so can't we just say 'Reset view zoom level to default', no 'its'?